cmd/kn-event/cmd/root.go

```diff
@@ -6,6 +6,7 @@ import (
 
 	"github.com/spf13/cobra"
 	"github.com/thediveo/enumflag"
+	_ "k8s.io/client-go/plugin/pkg/client/auth" // for kubeconfig auth plugins to work correctly see https://github.com/knative-sandbox/kn-plugin-event/issues/24
```

Comment:
I think some comments are required for fixing the lint error `File is not `gci`-ed (gci)` ?

Reply:
fixed